libs/components/src/lib/expanded-list/find-out-more.tsx

```diff
@@ -13,7 +13,7 @@ export function FindOutMore<T>(props: FindOutMoreProps<T>) {
       <button onClick={() => setIsOpen(!isOpen)}>
         {isOpen ? 'Close' : 'Find out more'}
       </button>
-      {isOpen && props.ExpandedComponent(props.ExpandedComponentProps)}
+      {isOpen && (props.ExpandedComponent as any)(props.ExpandedComponentProps)}
```

Comment:
I can't find an external Google TS style guide, but the internal one recommends not using `any`.  Use `unknown` if you can't be more specific.